be/src/agent/task_worker_pool.cpp

```diff
@@ -640,12 +748,12 @@ void* TaskWorkerPool::_publish_version_worker_thread_callback(void* arg_this) {
 
         std::vector<TTabletId> error_tablet_ids;
         uint32_t retry_time = 0;
-        Status res = Status::OK();
+        Status status;
+
         while (retry_time < PUBLISH_VERSION_MAX_RETRY) {
             error_tablet_ids.clear();
-            EnginePublishVersionTask engine_task(publish_version_req, &error_tablet_ids);
-            res = worker_pool_this->_env->storage_engine()->execute_task(&engine_task);
-            if (res.ok()) {
+            status = _publish_version_in_parallel(arg_this, threadpool, publish_version_req, &error_tablet_ids);
```

Comment:
I think we can get the num of tablets related in this publish version and add it to log in line 774

Reply:
Updated.